recipes-browser/cog/cog_0.16.0.bb

```diff
@@ -0,0 +1,16 @@
+require cog.inc
+require cog-meson.inc
+require conf/include/devupstream.inc
+
+DEFAULT_PREFERENCE = "-1"
+
+SRC_URI[sha256sum] = "fb91104e25e1dde27189c91c70acc356e387f47acebaa8997e01ce5879c3a600"
```

Comment:
This needs updating to 2956f719d36c9a6368f63901a0d1ceaf4d27f529f7df441b3748c36d72906fc6